src/blocks/accordion/edit.js

```diff
@@ -71,44 +70,42 @@ const AccordionEdit = ( props ) => {
 	const items = getBlocksByClientId( clientId );
 
 	const handleEvent = ( ) => {
-		if ( items[ 0 ].innerBlocks ) {
-			const lastId = items[ 0 ].innerBlocks[ items[ 0 ].innerBlocks.length - 1 ].clientId;
-			let copyAttributes = {};
-
-			if ( lastId ) {
-				const lastBlockClient = getBlockAttributes( lastId );
-				if ( lastBlockClient.backgroundColor ) {
-					copyAttributes = Object.assign( copyAttributes, {
-						backgroundColor: lastBlockClient.backgroundColor,
-					} );
-				}
-
-				if ( lastBlockClient.borderColor ) {
-					copyAttributes = Object.assign( copyAttributes, {
-						borderColor: lastBlockClient.borderColor,
-					} );
-				}
-
-				if ( lastBlockClient.textColor ) {
-					copyAttributes = Object.assign( copyAttributes, {
-						textColor: lastBlockClient.textColor,
-					} );
-				}
-
-				if ( lastBlockClient.customTextColor ) {
-					copyAttributes = Object.assign( copyAttributes, {
-						customTextColor: lastBlockClient.customTextColor,
-					} );
-				}
+		const lastId = items[ 0 ].innerBlocks[ items[ 0 ].innerBlocks.length - 1 ]?.clientId;
+		let copyAttributes = {};
+
+		if ( !! lastId ) {
+			const lastBlockClient = getBlockAttributes( lastId );
+			if ( lastBlockClient.backgroundColor ) {
```

Comment:
@aledesma-godaddy I know this is only a copy paste but I feel like we can utilize a loop here instead of multiple if statement. `['backgroundColor','textColor'].forEach( prop => { if...etc } )`

Reply:
Love the idea. I went ahead and made that change to the edit.js logic there.